core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1804,15 +1804,10 @@ private[spark] object Utils extends Logging {
 
   lazy val isInInterpreter: Boolean = {
```

Comment:
Let's keep this for now. I remember there's some discussion saying that this should be abstracted to a shared method so it can be used elsewhere.